packages/react-dev-utils/README.md

```diff
@@ -201,3 +201,35 @@ module.exports = {
   // ...
 }
 ```
+
+#### `new webpackEnvDefinePlugin({regex: string, customVariables: Object.<string, string>})`
+
+This webpack plugin grabs environment variables that follows a specified regex and injects them into the application 
+using the webpack DefinePlugin.
+In addition, the plugin allows defining custom environment variables to be injected.
+
+To use first reference it in your webpack config:
+
+  ```js
+  var EnvDefinePlugin = require('react-dev-utils/webpackEnvDefinePlugin');
+  ```
+
+and define it as a plugin in the webpack config plugin section:
+ 
+ ```js
+  plugins: [
+     ...
+     new EnvDefinePlugin({
+       // Grab MY_PREFIX_* environment variables
+       regex: /^MY_PREFIX_/i,
+       customVariables: {
+         // Useful for determining whether we’re running in production mode.
+         // Most importantly, it switches React into the correct mode.
```

Comment:
Both of these comments are a bit out of context (and the second one is actually misleading).
Can we come up with better realistic examples?